akd_core/src/configuration/mod.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+//
+// This source code is dual-licensed under either the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree or the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree. You may select, at your option, one of the above-listed licenses.
+
+//! Defines the configuration trait and implementations for various configurations
+
+mod traits;
+pub use traits::{Configuration, NamedConfiguration};
+
+// Note(new_config): Update this when adding a new configuration
+
+#[cfg(feature = "whatsapp_v1")]
```

Comment:
Even better than doing it at the crate root, import it under the common name here with the `as` syntax so people only need to update the configuration in 1 place and it'll take effect everywhere.

Reply:
See my above comment on features needing to be additive (so that enabling any combination of features should always compile / not break things)